From 23ba43d9aebb1f665135a41f0d36c5365b38c506 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Sebastian=20Dr=C3=B6ge?= <sebastian.droege@collabora.co.uk>
Date: Mon, 12 Mar 2012 13:04:41 +0100
Subject: [PATCH 11/22] Allow for a second distro GIO module path as used on
 Ubuntu

...because apparently only half of their packages are multi-arch
capable...
---
 gio/giomodule.c | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/gio/giomodule.c b/gio/giomodule.c
index 65e6c32..2557f74 100644
--- a/gio/giomodule.c
+++ b/gio/giomodule.c
@@ -1102,6 +1102,11 @@ _g_io_modules_ensure_loaded (void)
       g_io_modules_scan_all_in_directory_with_scope (GST_SDK_GLIB_GIO_DISTRO_GIO_MODULE_PATH, scope);
 #endif
 
+#ifdef GST_SDK_GLIB_GIO_DISTRO_GIO_MODULE_PATH2
+      /* Now load all modules from the distro, e.g. gvfs */
+      g_io_modules_scan_all_in_directory_with_scope (GST_SDK_GLIB_GIO_DISTRO_GIO_MODULE_PATH2, scope);
+#endif
+
       g_io_module_scope_free (scope);
 
       /* Initialize types from built-in "modules" */
-- 
1.8.4

